lib/ansible/modules/windows/win_service.py

```diff
@@ -59,6 +59,14 @@
       - restarted
     default: null
     aliases: []
+  delayed:
+    description:
+      - If True, when start_mode is C(auto) the service is flagged for a delayed start.
+    default: false
+    required: false
```

Comment:
I understand, we are trying to make things more concise over time. But at this time a lot of the modules are not consistent at various levels.